codechefcli/decorators.py

```diff
@@ -33,3 +33,32 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
 
     return wrapper
+
+
+def sort_it(func):
+    '''
+    desc: decorator method to sort the specified argument
+    '''
+    def wrapper(*args, **kwargs):
+        sort = args[0]
+        resp = func(*args, **kwargs)
+        if resp['code'] == 200 and resp['data_type'] == 'table':
+            data_rows = resp['data']
+            if sort:
+                heading = data_rows[0]
+                data_rows = data_rows[1:]
+                index = -1
+                if sort.upper() in heading:
+                    index = heading.index(sort.upper())
+                    data_rows.sort(key=lambda x: x[index])
+                    data_rows.insert(0, heading)
+                    resp['data'] = data_rows
+                else:
+                    data_rows.insert(0, heading)
```

Comment:
Isn't this unnecessary?

Reply:
I removed the heading, so needed to add it back in both the case if the sorting arg matched or not

Reply:
If we reach this `else` block, we do not need `data_rows` value.